### DISCUSSION TITLE
Angular 15

### DISCUSSION THREAD

**Discussion by nadirhadji** [mdbootstrap/TW-Elements#1431] [Support from community] [answered]
has anyone already integrated tailwind-elements to Angular? I need someone to tell me where to look

**Comment by Trochonovitz** [comment]
Hi, for installation purposes you got to download Tailwind Elements from NPM and then import library in app.component.ts file using lifecycle method ngOnInit - like this. After that create a new file called typings.d.ts in root directory and put there module declaration: declare module 'tw-elements'. FYI in next week we are going to release a bunch of tutorials for most popular CRS and SSR frameworks so stay tuned.
```
  ngOnInit() {
    const importTE = async () => {
      await import('tw-elements');
    };
    importTE();
  }
```

**Comment by filipkappa** [comment, marked as answer]
Here is an official Tailwind Elements + Angular integration guide:
https://tailwind-elements.com/docs/standard/integrations/angular-integration/

And here ou can sign up for the waiting list for early access to a dedicated Angular starter pack:
https://tailwind-elements.com/angular/
